build.sbt

```diff
@@ -1,21 +1,54 @@
+val circeVersion = "0.12.3"
+val circeDerivationVersion = "0.12.0-M7"
+val enumeratumVersion = "1.5.13"
+val enumeratumCirceVersion = "1.5.22"
+val scalaTestVersion = "3.1.0"
+
+val commonSettings = Seq(
```

Comment:
I moved to a multi-project layout so this project could still depend on itself without needing to pull in all the Jade-related stuff